content/blog/pragmatic-programming-lessons.md

```diff
@@ -0,0 +1,97 @@
++++
+authors = ["Anita Malhotra"]
+categories = ["Software Craftsmanship"]
```

Comment:
We can keep the same tags and categories, there's also a 'Book Review' tag that we have which will be apt for this